packages/app/app/plugins/stream/InvidiousPlugin.js

```diff
@@ -0,0 +1,69 @@
+import logger from 'electron-timber';
+
+import StreamProviderPlugin from '../streamProvider';
+import * as Invidious from '../../rest/Invidious';
+import YoutubePlugin from './YoutubePlugin';
+
+class InvidiousPlugin extends StreamProviderPlugin {
```

Comment:
I think I'll define these things properly in Typescript soon

Reply:
I was also thinking about moving reducer and action constants in @nuclear/core, without any electron reference (mobile app in mind).
What do you think about it ?

Reply:
We can still use reducers on mobile (I think) but decoupling electron dependencies is a step in a good direction